src/components/Expandable.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { ExpandableContainer, ExpandableTitle, ExpandableTrailingIcon } from './Expandable.style';
+import DownArrowIcon from './icons/DownArrowIcon';
+
+interface ExpandableProps {
+	title: string;
+	description: string;
+	expanded: boolean;
+	setExpanded: () => void;
+}
+
+export default function Expandable({ title, description, expanded, setExpanded }: ExpandableProps): JSX.Element {
+	return (
+		<ExpandableContainer>
+			<ExpandableTitle aria-label={expanded ? 'Hide answer' : 'Expand answer'} onClick={() => setExpanded()}>
+				{title}
+				<ExpandableTrailingIcon>
+					<DownArrowIcon />
```

Comment:
setExpanded() or some other mechanism should be responsible for changing this to an UpArrowIcon when expanded.

Reply:
[Updated](https://github.com/ardriveapp/price-calculator/commit/9268a6cb9e9bebfc75999c9e55fea8a36651668d)